### PR TITLE
Adding instrucment

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Adding `#[instrument]` to debug topdown syncer performance issue.

### CODE REVIEW

**Review by aakoshh — APPROVED** [review]
LGTM :+1:

**Review by aakoshh — CHANGES_REQUESTED** [review]
Sorry, just noticed that the tests are actually failing: https://github.com/consensus-shipyard/ipc/actions/runs/8098552876/job/22132258717

**Review by aakoshh — APPROVED** [review]
Ah, no worries, I see this is going into the other branch where you are already fixing them.

File: fendermint/app/src/main.rs [hunk 1]
```diff
@@ -44,6 +46,7 @@ fn init_tracing(opts: &options::Options) -> Option<WorkerGuard> {
             fmt::Layer::new()
                 .json()
                 .with_writer(non_blocking.with_max_level(log_level))
+                .with_span_events(FmtSpan::CLOSE)
```
[the remark this answers is not in the document]
Comment (fridrik01) [reply fendermint/app/src/main.rs:49]: The [documentation](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/struct.Layer.html#method.with_span_events) is a bit short in exact details, but I think if we use `FmtSpan::CLOSE` only like here then the generated event will also contain the total time the span was entered (+ time the span existed but was not entered).

**Review by fridrik01 — APPROVED** [review]
lgtm

**Pull request merged**